v22.1/follower-reads.md

```diff
@@ -1,197 +1,191 @@
+- Execute the `SET SESSION CHARACTERISTICS AS TRANSACTION AS OF SYSTEM TIME follower_read_timestamp()` [SQL statement](set-vars.html#special-syntax-cases). This has the same effect as setting the session variable as shown above.
 
-Any [`SELECT` statement](select-clause.html) with an appropriate [`AS OF SYSTEM TIME`](as-of-system-time.html) value can be an exact staleness follower read.
+You can set `default_transaction_use_follower_reads` on a per-role basis; for instructions, see [Set default session variable values for a role](alter-role.html#set-default-session-variable-values-for-a-role).
+{{site.data.alerts.end}}
 
-For exact staleness reads, you can use the convenience [function](functions-and-operators.html) `follower_read_timestamp()`, which returns a [`TIMESTAMP`](timestamp.html) that provides a high probability of being served locally while not [blocking on conflicting writes](#exact-staleness-reads-and-long-running-writes).
+### Bounded staleness reads
 
-Use this function in an `AS OF SYSTEM TIME` statement as follows:
+A _bounded staleness read_ is a historical read that uses a dynamic, system-determined timestamp to minimize staleness while being more tolerant to replication lag than an exact staleness read. Bounded staleness reads also help increase system availability, since they provide the ability to serve reads from local replicas even in the presence of network partitions or other failures that prevent the SQL gateway from communicating with the leaseholder.
 
-``` sql
-SELECT ... FROM ... AS OF SYSTEM TIME follower_read_timestamp();
-```
+#### When to use bounded staleness reads
 
-### Run queries that use bounded staleness follower reads
+Use bounded staleness follower reads when you:
 
-As described in [Bounded staleness read limitations](#bounded-staleness-read-limitations), bounded staleness follower reads are available under the following conditions:
+- Need minimally stale reads from the nearest replica without blocking on conflicting transactions. This is possible because the historical timestamp is chosen dynamically and the least stale timestamp that can be served locally without blocking is used.
+- Can confine the read to a single statement that meets the [bounded staleness limitations](#bounded-staleness-read-limitations).
+- Need higher availability than is provided by exact staleness reads. Specifically, what we mean by availability in this context is:
+  - The ability to serve a read with low latency from a local replica rather than a leaseholder.
+  - The ability to serve reads from local replicas even in the presence of a network partition or other failure event that prevents the SQL gateway from communicating with the leaseholder. Once a replica begins serving follower reads at a timestamp, it will always continue to serve follower reads at that timestamp. Even if the replica becomes completely partitioned away from the rest of its range, it will continue to stay available for (increasingly) stale reads.
```

Comment:
"network partitions" could link to https://www.cockroachlabs.com/docs/v21.2/cluster-setup-troubleshooting.html#network-partition

"replicas" could be the same link you used above, to the replica definition in the architecture docs 

I know this is repetitive but folks could land here directly via eg a link from Support